foxtrot-core/src/main/java/com/google/common/base/Stopwatch.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
```

Comment:
So basically StopWatch class from 18.0 is copied and constructor is added to make it 12.0 compatible ?